bee-snapshot/src/global/file.rs

```diff
@@ -37,11 +36,11 @@ pub enum Error {
 }
 
 impl GlobalSnapshot {
-    pub fn from_file(path: &str, index: MilestoneIndex) -> Result<Self, Error> {
+    pub fn from_file(path: &str, index: u32) -> Result<Self, Error> {
```

Comment:
Is it work putting a `type MilestoneIndex = u32;` in `bee-common`? I feel like a lot of these cross-crate types should be put in common with the crates themselves just defining the implementation.

Reply:
Yeah maybe... But that would be another PR, it touches too many things to be done now